schemas/README.md

```diff
@@ -1,39 +0,0 @@
-JSON Schema project: https://json-schema.org
-
-
-To validate json using https://github.com/Julian/jsonschema:
-
-Create pyenv:
-```
-export PYENV_ROOT=$HOME/pyenv
-export PATH=$PYENV_ROOT/bin:$PATH
-
-# pyenv install 3.7.4 # uncomment for the first time
-export PYENV_VERSION=3.7.4
-
-eval "$(pyenv init -)"
-```
-
-Install jsonschema:
-```
-pip install jsonschema
-```
-
-To validate JSON with schema just load it with python:
```

Comment:
This is the only file I was not sure what to do with it: in one hand when reading it I felt it was not that useful (we are already providing a method to validate a file with a JSON schema) and there is no information about how to produce such schemas (which I thought that is what it was about), but on the other if it was there is because maybe it was useful for some of you. I'm happy to bring it back in if requested.

Reply:
I believe if someone wants to use json schema to validate files that we do not support here, then they just need to look at the Python json schema documentation itself? No need to do that ourselves here.

Reply:
for me this was a useful hack I'm always forgetting how to run.
M.b. we can point to our own validator script usage. Or just have it on our docs  🙈